### ISSUE TITLE
[Request]: Option to Remove Desktop Entry

### ISSUE THREAD

**Issue by AsciiWolf** [bottlesdevs/Bottles#2256]
### Tell us the problem or your need

There is an "Add Desktop Entry" menu item for installed applications. However, I did not found any way to remove these entries using Bottles after they are added. It would be great if the "Add Desktop Entry" menu item changed to "Remove Desktop Entry" if the desktop menu entry already exist. The same could be implemented for Steam entries.

### Describe the solution you'd like

Detect if there is already an application desktop entry created by Bottles, if so, allow removing such entry by right clicking the application in Boxes and selecting "Remove Desktop Entry".

**Comment by ProfessorCode212** [comment]
I agree. It would also be useful if Bottles automatically removes the desktop entry if an application gets uninstalled.

**Comment by Perkolator** [comment]
> I agree. It would also be useful if Bottles automatically removes the desktop entry after an application gets uninstalled.

Or if the whole bottle gets deleted from Bottles GUI.

**Comment by Perkolator** [comment]
New close-bot mass closing open issues?! This is still a valid feature request.
